tests/bot-entrypoint.py

```diff
@@ -24,11 +24,14 @@ def load_env():
         "bin/keybaseca --wipe-all-configs\n"
         "bin/keybaseca --wipe-logs || true\n"
         "bin/keybaseca generate --overwrite-existing-key\n"
-        "echo yes | bin/keybaseca backup > /shared/cakey.backup\n"
+        # The output from this backup is tested in test_env_1.py
+        "echo yes | bin/keybaseca backup > /mnt/cakey.backup\n"
```

Comment:
can't say i love this pattern. i guess there's no way to trigger this from outside, and that's kinda the point. 🤷‍♂